### PR TITLE
feature/819

### LINKED ISSUE #819: New Airport: KRDU (Raleigh-Durham Int'l)
Raleigh–Durham International Airport is the main airport serving Raleigh, Durham, and the surrounding Research Triangle region of North Carolina. It is located 4 1⁄2 miles northeast of the town of Morrisville in Wake County. The airport covers 5,000 acres and has three runways and has passenger service to 50 destinations with 400 average daily departures, including nonstop intercontinental service to London and Paris.

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #819 

Adds KRDU

### Completion Checklist
- [x] terrain file
- [x] polygonal airspace boundary
- [x] all SIDs
- [x] all STARs
- [x] information to validate traffic levels
- [x] base video map
- [x] no console errors or warnings
- [x] airport guide
---

According to Wikipedia, KRDU has 191,348 yearly operations. Traffic level should fall between 21.8acph and 65.5acph. Currently, there's a total of 23.2acph 👍

### CODE REVIEW

File: assets/airports/krdu.json [hunk 1]
```diff
@@ -0,0 +1,4178 @@
...
+                ["swa", 25],
+                ["ual", 9],
+                ["vrd", 1]
+            ]
+        },
+        {
+            "origin": "",
+            "destination": "KRDU",
+            "category": "arrival",
+            "route": "ROA.ALDAN1.KRDU",
```
Comment (erikquinn) [inline assets/airports/krdu.json]: needs runways in all procedure entries/exits

**Comment by Fechulo** [comment]
All ready!

File: assets/airports/krdu.json [hunk 2]
```diff
@@ -0,0 +1,4178 @@
...
+            "body": ["PEAVY", "OXFRD", "HARVY", "CARML"],
+            "exitPoints": {
+                "FAK": ["FAK"]
+            },
+            "draw": [
+                ["PEAVY", "OXFRD", "HARVY", "CARML", "FAK*"]
+            ]
+        },
+        "PACK7": {
+            "icao": "PACK7",
```
Comment (indianbhaji) [inline assets/airports/krdu.json]: `PACKK7`

File: assets/airports/krdu.json [hunk 3]
```diff
@@ -0,0 +1,4178 @@
...
+            },
+            "body": ["PEAVY", "OXFRD", "HARVY", "CARML"],
+            "exitPoints": {
+                "FAK": ["FAK"]
+            },
+            "draw": [
+                ["PEAVY", "OXFRD", "HARVY", "CARML", "FAK*"]
+            ]
+        },
+        "PACK7": {
```
Comment (indianbhaji) [inline assets/airports/krdu.json]: `PACKK7`

File: assets/airports/krdu.json [hunk 4]
```diff
@@ -0,0 +1,4178 @@
...
+                ["FKN*", "DUFFI", "MEYER", "ARGAL", "RDU"],
+                ["SBY*", "CCV", "DRIVE", "FKN"],
+                ["HPW*", "DUFFI"],
+                ["SWL*", "CCV"]
+            ]
+        },
+        "BLOGS1": {
+            "icao": "BLOGS1",
+            "name": "Blogs One",
+            "entryPoints": {
```
Comment (indianbhaji) [inline assets/airports/krdu.json]: Alphabeticalise `entryPoints`

File: assets/airports/krdu.json [hunk 5]
```diff
@@ -0,0 +1,4178 @@
...
+                ["ISO*", "TILTZ", "BLOGS", "CABED", "AGOLE", "SULIE", "TIBIA", "UNCUT"],
+                ["CABED", "ORLAC", "TIIME", "STINT", "TOYER"],
+                ["TYI*", "ARELE", "BLOGS"],
+                ["ILM*", "TILTZ"]
+            ]
+        },
+        "BRADE5": {
+            "icao": "BRADE5",
+            "name": "Brady Five",
+            "entryPoints": {
```
Comment (indianbhaji) [inline assets/airports/krdu.json]: Alphabeticalise `entryPoints`

File: documentation/airport-guides/krdu.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+Raleigh-Durham Combined TRACON and ATCT (RDU), Morrisville, North Carolina, USA
+257,412 annual operations in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Tracon.asp))
+
+This level 9 combined TRACON/ATCT facility controls a number of airfields in Greater Raleigh. Since this is a combined facility, the TRACON and ATC Tower are co-located at Raleigh-Durham International Airport (KRDU).
+
+### Airspace
+The RDU TRACON airspace encompasses a large area around Great Raleigh extending from the surface up to 12,000 feet. Class C airspace exists with tops of 4,400 feet over the Raleigh and Durham areas.
+
+### Instrument Procedures
+Although departing aircraft must be vectored to their initial fixes, the STAR's are very automated, taking a lot off weight off the controllers. Arrivals must still be vectored to their final approach course.
```
Comment (indianbhaji) [inline documentation/airport-guides/krdu.md]: the STARs are very automated (without the apostrophe)
taking a lot of weight off (extra f)

File: documentation/airport-guides/krdu.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+203,174 annual operations
+
+Located between the cities of Raleigh and Durham, RDU is the main airport in the area. Originally built as an Air Force base, its old runways are still visible on the southeast side of the airport. It now has three runways, two of which are parallel (5L/R - 23L/R). Simultaneous approaches are permitted into these runways although it is rarely necessary due to the low amounts of traffic. 
+
+#### KTTA
+Raleigh Executive Jetport  
+63,000 annual operations  
+23.4nm Southwest of KMCI
+
+Officially called "Raleigh Exec: The Raleigh Executive Jetport @ Sanford-Lee County", the airfield specializes in corporate and recreational flights into the Research Triangle Region
```
Comment (indianbhaji) [inline documentation/airport-guides/krdu.md]: Full stop

File: documentation/airport-guides/krdu.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+#### KLHZ
+Triangle North Executive  
+62,800 annual operations  
+23.9nm Northeast of KMCI
+
+Also known as Franklin County Airport, Triangle North Executive is used primarily for private and general aviation.
+
+### Additional Resources
+[VATSIM SOP](https://vzdc.org/downloads/sops/RDU.pdf)
```
Comment (indianbhaji) [inline documentation/airport-guides/krdu.md]: Charts maybe? https://nfdc.faa.gov/nfdcApps/services/ajv5/airportDisplay.jsp?airportId=krdu

File: assets/airports/krdu.json (hunk 2 above)
Comment (Fechulo) [reply assets/airports/krdu.json]: The ICAO code for the procedure is actually `PACK7` as weird as that is:
pack

**Pull request merged**